src/app/routes/applications/new-role/components/role-name/role-name.component.html

```diff
@@ -19,7 +19,7 @@
             type="text"
             placeholder="Role Name"
             required
-            oninput="this.value = this.value.toLowerCase()"
+            oninput="value = value.toLowerCase().replace(/\s+/g, '')"
```

Comment:
This case is repeating, maybe there is a way to make it more generic across app?